### PR TITLE
Retail documentation rework including containerized retail

### PR DESCRIPTION
**Pull request by aaannz**
# Description

Port of https://github.com/uyuni-project/uyuni-docs/pull/1574

Rework retail documentation to cover different configuration ways by including example configurations.

Include instructions how to configure retail using containerized proxy in the examples.

# Target branches

Which documentation version does this PR apply to?

- [ ] Master (Default)  https://github.com/uyuni-project/uyuni-docs/pull/1574
- [X] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes https://github.com/SUSE/spacewalk/issues/19844

### CODE REVIEW

File: modules/retail/pages/containerized-saltboot.adoc [hunk 1]
```diff
@@ -0,0 +1,182 @@
+[[retail-install-setup-containerized]]
+= Set Up the {productname} {smr} Environment using containerized proxy
+
+To set up the {productname} {smr} environment, you will need to have already installed and configured:
+
+* {productname} {smr} Server 4.3 or newer
+* one or more {productname} {smr} containerized proxies
+* one or more {productname} build host
```
Comment (keichwa) [inline modules/retail/pages/containerized-saltboot.adoc:8]:
```suggestion
* one or more {productname} build hosts
```

File: modules/retail/pages/containerized-saltboot.adoc [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+Saltboot Groups are branch groups, system group with branch id as its name, with [systemitem]``Saltboot Group`` formula enabled.
+
+image::saltboot_group.png[scaledwidth=80%]
+
+[systemitem]``Saltboot Group`` formula is a successor of [systemitem]``Branch Network formula``, [systemitem]``PXE formula`` and [systemitem]``TFTP formula`` used in regular {productname} {smr} setups.
+
+Name of the [systemitem]``Saltboot group`` is automatically used as a [systemitem]``branch id``, an identifier for group of machines booted through particular containerized {productname} Proxy.
+
+All [systemitem]``saltboot`` deployed machines though containerized proxy will automatically became members of its [systemitem]``Saltboot group``.
```
Comment (keichwa) [inline modules/retail/pages/containerized-saltboot.adoc:67]:
```suggestion
All [systemitem]``saltboot`` deployed machines though containerized proxy will automatically become members of its [systemitem]``Saltboot group``.
```

**Review by keichwa — CHANGES_REQUESTED** [review]
I started with the review, here some typo fixes.  I'll continue later (or next week).

**Comment by aaannz** [comment]
These are cherry-picked commits directly from https://github.com/uyuni-project/uyuni-docs/pull/1574

**Review by keichwa — APPROVED** [review]
Thanks for pointing out!  In general it LGTM!

**Review by jcayouette — APPROVED** [review]

**Pull request merged**